README.adoc

```diff
@@ -212,13 +212,14 @@ include::finish/inventory/Containerfile[]
 
 The [hotspot=from file=0]`FROM` instruction initializes a new build stage, which indicates the parent image of the built image. If you don't need a parent image, then you can use `FROM scratch`, which makes your image a base image. 
 
-In this case, you're using the recommended production image from the IBM container registry,
-`icr.io/appcafe/open-liberty:full-java11-openj9-ubi`, as your parent image. If you
-don't want any additional runtime features for your `kernel` image, define the
-`FROM` instruction as `FROM icr.io/appcafe/open-liberty:kernel-java11-openj9-ubi`. To use the default image that
-comes with the Open Liberty runtime, define the `FROM` instruction as `FROM icr.io/appcafe/open-liberty`. 
-You can find all the https://hub.docker.com/_/open-liberty[official images^] and
-https://hub.docker.com/r/openliberty/open-liberty/[ubi images^] on the open-liberty Docker Hub.
+In this guide, you're using the production image from the IBM Container Registry (ICR),
```

Comment:
Suggestion:
```adoc
In this guide, you're using an official image from the IBM Container Registry (ICR),
`icr.io/appcafe/open-liberty:full-java11-openj9-ubi`, as your parent image. This image is tagged with the word `full`, meaning it includes all Liberty features. `full` images are recommended for development only since they significantly expand the image size with features that are not required by the application.

In production, to minimize your image footprint, it is recommended that you use one of the `kernel-slim` images. For example, `icr.io/appcafe/open-liberty:kernel-slim-java11-openj9-ubi`.
This will install the basic server, to which you can then add all the necessary features for your application via the usage pattern detailed in the Open Liberty https://github.com/OpenLiberty/ci.docker#building-an-application-image[container image documentation^].
To use the default image that comes with the Open Liberty runtime, define the `FROM` instruction as `FROM icr.io/appcafe/open-liberty`. 
You can find all official images on the Open Liberty https://github.com/OpenLiberty/ci.docker#container-images[container image repository^].
```

Reply:
Thank you, implemented in latest commit.